src/lib/operators/limit.cpp

```diff
@@ -26,7 +26,8 @@ std::shared_ptr<AbstractExpression> Limit::row_count_expression() const { return
 
 std::shared_ptr<AbstractOperator> Limit::_on_deep_copy(
     const std::shared_ptr<AbstractOperator>& copied_left_input,
-    const std::shared_ptr<AbstractOperator>& copied_right_input) const {
+    const std::shared_ptr<AbstractOperator>& copied_right_input,
+    std::unordered_map<const AbstractOperator*, std::shared_ptr<AbstractOperator>>& copied_ops) const {
   return std::make_shared<Limit>(copied_left_input, _row_count_expression->deep_copy());
```

Comment:
Same as for the LQP expressions. I have not yet understood in which cases we forward `copied_ops` and in which we don't.